src/app/components/forms/AddressWithValidation.tsx

```diff
@@ -81,35 +81,62 @@ export default function AddressWithValidation<
 
   const addressSuggestions = useAddressSuggestion();
 
+  const [hasInteracted, setHasInteracted] = React.useState(false);
```

Comment:
```suggestion
  const [hasInteracted, setHasInteracted] = useState(false);
```
On peut importer `useState` dans ce fichier histoire d'alléger l'écriture